server.js

```diff
@@ -0,0 +1,40 @@
+// Require express module and create app.
+var express = require('express');
+var app = express();
+
+// Require mongoose and connect to mongodb.
+var mongoose = require('mongoose');
+var db = mongoose.createConnection('localhost', 'single_resource_rest_api');
+
+// Handle db connect errors.
+db.on('error', function() { throw new Error('Failed to connect to the database.'); });
+
+// Handle db connect success.
+db.once('open', function() {    
+    
+    // Load bus schema and create model. Get a list of buses and output as JSON.
+    app.get('/bus', function(req, res) {
```

Comment:
There needs to be a post, put, and delete request for this assignment. I suggest following Tyler's code as an example to go by.